packages/table/src/FzTable.vue

```diff
@@ -0,0 +1,277 @@
+const headerClasses = computed(() => {
+
+})
+
+const bodyStaticClasses = ['fz__body', 'z-[1]', 'px-16', 'h-48', 'bg-core-white', 'flex', 'justify-start', 'items-center', 'min-w-min', 'border-b-1', 'border-grey-100']
+const bodyClasses = computed(() => {
+
+})
+
+const getBodyClasses = (column: {props: FzColumnProps, children: FzColumnSlots}, isHeader?: boolean) => {
+  return {
+    relative: !column.props.sticky,
+    sticky: column.props.sticky,
+    'left-0 z-[2]': column.props.sticky === 'left',
+    'z-[3]': column.props.sticky && isHeader,
+    'right-0': column.props.sticky === 'right'
+  }
+}
+
+const updateClasses = () => {
+  const tableWidth = columns.reduce((acc, el, index) => {
+    const colWidth = grid.value.children[index].getBoundingClientRect().width
+    acc+=colWidth
+    return acc
+  }, 0)
+  grid.value.style.width = `${tableWidth}px`
+}
+
+const centerPageList = computed(() => {
+  if (!props.pages) {
+    return []
+  }
+  const safeActivePage = activePage.value || 0
+  return new Array(props.pages)
+    .fill(0)
+    .map((val, index) => index)
+    .filter(el => {
+      const res = (Math.abs(el - safeActivePage) <= props.pageInterval) && el !== 0 && el !== props.pages - 1
+      return res
+    })
+})
+
+onMounted(() => {
+  updateClasses()
+
+  if (props.actions?.items.length) {
+
+  }
+})
+onUpdated(updateClasses)
+</script>
+
+<template>
+<div class="fz__table overflow-auto size-full">
+  <div
+    :class="[staticClasses, classes]"
+    ref="grid"
+    role="table"
+    :aria-rowcount="value.length"
+    :aria-colcount="columns.length">
+    <template  v-for="(column, index) in columns">
+      <div :class="[headerStaticClasses, headerClasses, getBodyClasses(column, true)]" role="columnheader" aria-sort="none" >
+         {{ column.props.header }} 
+      </div>
+    </template>  
+    <div
+      v-if="actions"
+      :class="['fz__table__header--actions', headerStaticClasses, headerClasses]">
+      Azioni
+    </div>
+    <template v-if="value && value.length" v-for="(row, index) in value">
+      <div
+        :aria-rowindex="index"
+        :class="[bodyStaticClasses, bodyClasses, getBodyClasses(column)]"
+        role="cell"
+        v-for="column in columns">
+        {{ row[column.props.field?.toLowerCase() || column.props.header.toLowerCase()] }}
+      </div>
+      <div
+        v-if="actions"
+        :class="[bodyStaticClasses, bodyClasses]">
+        <FzIconDropdown :actions="actions.items"></FzIconDropdown>
+      </div>
+    </template>
+    <div :class="['fz__table__empty', 'h-full mt-80', 'justify-self-center', colspan]" v-else>
+      {{ placeholder ?? "No data available" }}
+    </div>
+  </div>
+  
+</div>
+<div class="fz__table__footer w-full flex flex-row justify-end m-8" v-if="pages && value?.length">
+  <div class="fz__table__pagination flex flex-row justify-between items-center gap-8">
```

Comment:
I would create a separate componente called `FzTabelPagination`: it will be easier to refactor/move to another package when the designers will create a Pagination component